requirements-jammy.txt

```diff
@@ -1 +1 @@
-python-apt @ https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/python-apt/2.4.0ubuntu3/python-apt_2.4.0ubuntu3.tar.xz ; sys_platform == "linux"
+python-apt @ https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/python-apt/2.4.0ubuntu4/python-apt_2.4.0ubuntu4.tar.xz ; sys_platform == "linux"
```

Comment:
when are you going to join the cool guys over at core24??

Reply:
Probably the same time we drop CentOS 7 and Windows support. So.. Soon, I hope?